manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```diff
@@ -54,3 +54,11 @@ vars:
 
 configurations:
   - params.yaml
+
+configMapGenerator:
+  - name: katib-config
+    behavior: replace
+    files:
+      - katib-config.yaml
+    options:
+      disableNameSuffixHash: true
```

Comment:
Do we need it for Kubeflow if we re-use the `katib-cert-manager` install for Kubeflow?

Reply:
It makes sense. We should remove this configMapGenerator. Thanks.

Reply:
done.